tests/test_acasclient.py

```diff
@@ -449,6 +453,92 @@ def experiment_load_test(self, data_file_to_upload, dry_run_mode, expect_failure
             self.assertIsNone(response['transactionId'])
         else:
             self.assertIsNotNone(response['transactionId'])
+            self.assertIsNotNone(response['results']['experimentCode'])
+            experimentCode = response['results']['experimentCode']
+            experiment_dict = self.client.get_experiment_by_code(experimentCode)
+            experiment = Experiment(experiment_dict, self.client)
+
+            # Make sure the file was moved to the experiment folder by checking the file path
+            self.assertIn(experimentCode, experiment.source_file)
+
+            # Get the experiment source file
+            # Compare the source file['content'] to the data_file_to_upload content
+            # If the file['content'] is bytes then read the data_file_to_upload as bytes
+            # If the file['content'] is string then read the data_file_to_upload as string
+            source_file = experiment.get_source_file()
+            if isinstance(source_file['content'], bytes):
```

Comment:
This little block of code is repeated a few times. Do you want to make it into a helper function?